Bottom: 1dea80a703036b8141e234ea76f2cf07059cf89f
Top:    f6db665131989166508d7508428282697dffd14a
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-01-03 02:41:42 +0000

Refresh of configure-ac-save-and-restore--0

---

diff --git a/configure.ac b/configure.ac
index 8ed21a0..7e8348b 100644
--- a/configure.ac
+++ b/configure.ac
@@ -791,8 +791,10 @@ you will get lucky.
 	# since some linux systems evidently install gdlib-config but fail to
 	# install the headers (nice), check for the header too and fail if it
 	# is not there.
+	save_CPPFLAGS="$CPPFLAGS"
 	CPPFLAGS="$CPPFLAGS $GD_CFLAGS"
 	AC_CHECK_HEADERS(gd.h)
+	CPPFLASS="$save_CPPFLAGS"
 	case $ac_cv_header_gd_h in
 	     no )
 		AC_MSG_ERROR([
@@ -1105,7 +1107,7 @@ AC_DEFINE_UNQUOTED([COORD_MAX],[$COORD_MAX],
 
 # ------------- Complete set of CFLAGS and LIBS -------------------
 
-CFLAGS="$CFLAGS $X_CFLAGS $DBUS_CFLAGS $GLIB_CFLAGS $GTK_CFLAGS $CAIRO_CFLAGS $GTKGLEXT_CFLAGS $GLU_CFLAGS $GL_CFLAGS"
+CFLAGS="$X_CFLAGS $DBUS_CFLAGS $GLIB_CFLAGS $GTK_CFLAGS $GD_CFLAGS $CAIRO_CFLAGS $GTKGLEXT_CFLAGS $GLU_CFLAGS $GL_CFLAGS"
 LIBS="$LIBS $XM_LIBS $DBUS_LIBS $X_LIBS $GLIB_LIBS $GTK_LIBS $DMALLOC_LIBS $GD_LIBS $INTLLIBS $CAIRO_LIBS $GTKGLEXT_LIBS $GLU_LIBS $GL_LIBS"
